src/main/java/com/segment/analytics/android/integrations/flurry/FlurryIntegration.java

```diff
@@ -172,11 +171,5 @@ public void identify(IdentifyPayload identify) {
     }
 
     AnalyticsContext.Location location = identify.context().location();
```

Comment:
Can probably remove this line b/c it seems `location` is only being used in the deleted code below.